pkg/plugin/v3/scaffolds/api.go

```diff
@@ -56,6 +56,7 @@ func NewAPIScaffolder(
 	doResource, doController bool,
```

Comment:
all the changes made in this file are not required at all by following the suggestion made in https://github.com/kubernetes-sigs/kubebuilder/pull/1644/files#r510025737 and can be reverted.